geonames/geonames-waypointy.php

```diff
@@ -65,20 +26,17 @@
 
     echo "$waypoint, $lat, $lon::  ";
 
+    // get from geo.kumy.org
     $country=get_country_from_coords($lat, $lon);
     //print_r($country);
     echo "[ $country] ";
 
 
     if ($country == '' or $country == 'xyz') {
-        $handle = fopen("http://ws.geonames.org/countryCode?lat=$lat&lng=$lon&radius=30", "rb");
-        $country = strtolower(trim(fread($handle, 3)));
-        echo "geonames...";
-        fclose($handle);
+        $country = geonames_country_from_coords($geonamesEndpoint, $geonamesUsername, $lat, $lon);
         sleep(2);
     }
 
-
     if (($country == '<!d') or ($country == '')) {
         $url="http://maps.google.com/maps/geo?q=$lat,$lon&output=json&sensor=false";
```

Comment:
This service is dead (or has changed url)?

Reply:
I will check that point but remember that one google case were working.. will see